client/my-sites/stats/stats-purchase/stats-purchase-personal.tsx

```diff
@@ -85,33 +90,35 @@ const PersonalPurchase = ( {
 		);
 	} ) as RenderThumbFunction;
 
-	const handleClick = ( e: React.MouseEvent< HTMLAnchorElement, MouseEvent > ) =>
-		handlePlanSwap( e );
-
-	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
 	const isWPCOMSite = useSelector( ( state ) => siteId && getIsSiteWPCOM( state, siteId ) );
 	// The button of @automattic/components has built-in color scheme support for Calypso.
 	const ButtonComponent = isWPCOMSite ? CalypsoButton : Button;
-
-	const isTierUpgradeSliderEnabled = config.isEnabled( 'stats/tier-upgrade-slider' );
-	const handleSliderChanged = ( index: number ) => {
-		// TODO: Remove state from caller.
-		// Caller expects an index but doesn't do anything with it.
-		// Value is used below to determine tier price.
-		setSubscriptionValue( index );
-	};
-
 	// TODO: Remove old slider code paths.
 	const showOldSlider = ! isTierUpgradeSliderEnabled;
-	const isNewPurchaseFlowEnabled = config.isEnabled( 'stats/checkout-flows-v2' );
 
 	let continueButtonText = isStandalone
 		? translate( 'Get Stats' )
 		: translate( 'Get Jetpack Stats' );
+
 	if ( isNewPurchaseFlowEnabled ) {
 		continueButtonText = translate( 'Contribute and continue' );
 	}
 
+	const { mutateAsync: mutateNoticeVisbilityAsync } = useNoticeVisibilityMutation(
+		siteId,
+		'focus_jetpack_purchase'
+	);
```

Comment:
Do we plan to skip the purchase page permanently or postpone it for a while?

Reply:
I don't think we defined this anywhere - I'm thinking maybe we only mute it for like a month?

Reply:
Ok, I'll add a ticket for that and we and apply the same logic we used for the usage notice.